### PR TITLE
typography issue solved

### PR DESCRIPTION
**Pull request by chintan0513**
- [x] I have read and agree to the [Code of Conduct](https://github.com/danielschuster-muc/potter-db/blob/master/CODE_OF_CONDUCT.md), and have followed the [Contributing guidelines](https://github.com/danielschuster-muc/potter-db/blob/master/CONTRIBUTING.md).

## Pull Request Type

- [x] Bug fix ()
## Before
Screenshot (7)

## After
Screenshot (8)

## Summary
Added CSS property in Link tag

### CODE REVIEW

**Comment by danielschuster-muc** [comment]
Thank you for your PR. Adding the `textDecoration: none` is definitely much better. 

As you are currently working on the Navbar, could you maybe change the icon to the [potter db icon](https://github.com/danielschuster-muc/potter-db/blob/master/frontend/public/android-chrome-192x192.png)? 
For that I would suggest converting the image to svg and importing the svg with the [next/image](https://nextjs.org/docs/api-reference/next/image) component. 

And can you please check the checkboxes in the pr description.

**Comment by chintan0513** [comment]
Thank you for your response. 

Sure, I would be happy to work on this portion as well as react part.

Check ✔️

**Comment by danielschuster-muc** [comment]
Great, thank you!

**Comment by chintan0513** [comment]
Icon*

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
I think you need to add the icon itself.

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Can you please change the icon to [this one](https://github.com/danielschuster-muc/potter-db/blob/master/frontend/public/android-chrome-192x192.png) by converting it to a svg icon?

**Comment by danielschuster-muc** [comment]
No, it was fine how it was, just replace the `images/logo.svg` icon, which you added and is currently a broomstick, with [this icon](https://github.com/danielschuster-muc/potter-db/blob/master/logo.svg). And leave the rest as it was before [this commit](https://github.com/danielschuster-muc/potter-db/pull/217/commits/61fb04d673678e3299dd1cadf189b988383a0667).

File: frontend/next.config.js [hunk 1]
```diff
@@ -3,7 +3,9 @@ const nextConfig = {
   reactStrictMode: true,
   swcMinify: true,
   images: {
-    domains: ["static.wikia.nocookie.net", "www.wizardingworld.com"],
+    dangerouslyAllowSVG: true,
+    contentSecurityPolicy: "default-src 'self'; script-src 'none'; sandbox;",
```
Comment (danielschuster-muc) [inline frontend/next.config.js:7]: This is not required.

File: frontend/src/components/layout/Navbar.js [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 const Navbar = () => {
   return (
     <AppBar component="nav" color="transparent" elevation={0} position="static">
       <Toolbar disableGutters>
-        <Link href="/">
+        <Link href="/" style={{ textDecoration: 'none' }}>
           <Box display="flex">
-            <Bolt fontSize="large" />
+          <Image src="/images/logo.svg" width={30} height={10} alt="logo" />
```
Comment (danielschuster-muc) [inline frontend/src/components/layout/Navbar.js:13]: why do you have a different width and height for a square icon?

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Please just **copy** the icon located on https://github.com/danielschuster-muc/potter-db/blob/master/logo.svg. 
And don't move or rename any files like frontend/public/android-chrome-12x192.png.

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
Looks good so far. 
But you accidentially renamed `android-chrome-192x192.png` to `android-chrome-12x192.png`.

**Comment by chintan0513** [comment]
What about now?

**Comment by chintan0513** [comment]
Sure I'll try and let you know

**Review by danielschuster-muc — CHANGES_REQUESTED** [review]
No, don't delete `frontend/public/android-chrome-192x192.png`, it's important.

**Review by danielschuster-muc — APPROVED** [review]
Everything looks good! Thank you for contributing to Potter DB.

**Pull request merged**